lib/class-wp-rest-widget-areas-controller.php

```diff
@@ -84,6 +84,53 @@ public function register_routes() {
 		);
 	}
 
+	/**
+	 * Retrieves the comment's schema, conforming to JSON Schema.
+	 *
+	 * @since 4.7.0
+	 *
+	 * @return array
+	 */
+	public function get_item_schema() {
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => 'Widget Area',
```

Comment:
```suggestion
			'title'      => 'widget-area',
```

WordPress deviates from the JSON Schema spec here and uses it more like the `id` field. See the block renderer controller.